username@hostname:~/python-testing-example$ ./run
Testing Example - 2017-08-05 12:16:31.984962
test_make_something (test_AnExample.TestAnExample) ... ok
test_report_something (test_AnExample.TestAnExample) ... ok
test_should_fail (test_AnExample.TestAnExample) ... FAIL

======================================================================
FAIL: test_should_fail (test_AnExample.TestAnExample)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/lathama/git/python-testing-example/testingdemo/unittests/test_AnExample.py", line 28, in test_should_fail
    self.assertTrue(self.theclass.should_fail())
AssertionError: False is not true

----------------------------------------------------------------------
Ran 3 tests in 0.002s

FAILED (failures=1)
 tests in 0.002s

FAILED (errors=1)


Currently epoch today and 24 hours ago are: 1501953391.9934812 and 1501866991.9934812
Code coverage report done. See the index.html in: 
        /coverage
Style Check Start
./testingdemo/__init__.py:36:1: E402 module level import not at top of file
./testingdemo/__init__.py:42:80: E501 line too long (90 > 79 characters)
./testingdemo/howto.py:5:1: E302 expected 2 blank lines, found 1
./testingdemo/howto.py:19:80: E501 line too long (98 > 79 characters)
./testingdemo/howto.py:25:1: E265 block comment should start with '# '
./testingdemo/howto.py:25:35: W292 no newline at end of file
./testingdemo/unittests/test_AnExample.py:8:1: E302 expected 2 blank lines, found 1
./testingdemo/unittests/test_AnExample.py:23:67: W291 trailing whitespace
./testingdemo/unittests/test_AnExample.py:29:1: W293 blank line contains whitespace
./testingdemo/unittests/test_AnExample.py:30:1: E265 block comment should start with '# '
./testingdemo/unittests/test_AnExample.py:30:46: W292 no newline at end of file
Style Check End

The testingdemo version is: 1.0.1
